replay.js

```diff
@@ -283,9 +283,42 @@ async function main() {
         }
     });
 
+    /** @type {import("typescript/lib/protocol").CompletionInfoResponse | undefined} */
```

Comment:
IDK if we're already using it (have not expanded the diff), but we should change to use `ts.server.protocol` here given I eliminated `/lib/protocol` in 5.0.